html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>
+      <a href="../index.html">Home Page</a>
+    </li>
+    <li>
+      <a href="../about-me/about-me.html">About Me</a>
```

Comment:
In this case, you are exiting the current directory just to navigate right back into it, that is unnecessary. Instead, you can simply state the name of the file you want to show.